### PR TITLE
Add page to symptom surveys block to document contingency tables

### PR DESCRIPTION
**Pull request by nmdefries**
Adds documentation for contingency tables. This is largely taken from the existing README, plus some formatting and intro. This page is nested under `COVID Symptom Survey`, between `Response Files` and 'Survey Weights`.

Adds information about this data source and aggregates in the API to the `data-access` page.

### CODE REVIEW

File: docs/symptom-survey/data-access.md [hunk 1]
```diff
@@ -12,11 +12,16 @@ spread of COVID-19 and its effects on public health and well-being. This may
...
 access](https://dataforgood.fb.com/docs/covid-19-symptom-survey-request-for-data-access/),
 which sets out the basic conditions and provides a form to request access. An
 [international version of the COVID Symptom Survey](https://covidmap.umd.edu/)
 is conducted by the University of Maryland (UMD) and access can be requested
 through the same form.
+
+High-level aggregates of select survey items are available in the [COVIDcast
+API](../api/fb-survey.md), more information [here](../api/covidcast.md). Finer aggregates grouped by
+various demographic characteristics are available for download, more information
+[here](./contingency-tables.md).
```
Comment (krivard) [inline docs/symptom-survey/data-access.md:27]: drive-by comment -- if we want people with screen readers to be able to use our data, we should make sure the link text is descriptive. Links often just get listed out all together after the body text of the page has been read, and nobody will be able figure out which instance of "here" goes to which document.

Relevant W3C criteria:
* https://www.w3.org/TR/WCAG21/#link-purpose-in-context
* https://www.w3.org/TR/WCAG21/#link-purpose-link-only

**Comment by capnrefsmmat** [comment]
I went ahead and made some edits directly. [Preview of rendered version available here](http://rosmarus.refsmmat.com/delphi-epidata/symptom-survey/contingency-tables.html) (note: link will change, do not share, etc.).

I think it looks pretty good now. Once we're ready to go live, we can merge this and announce the new aggregates.

**Comment by nmdefries** [comment]
Taking a look.

File: docs/symptom-survey/contingency-tables.md [hunk 2]
```diff
@@ -14,9 +14,13 @@ demographic features:
 * [Weekly files](https://cmu.box.com/s/xwjulq0pteen52d4upni9ikagu7d8bl2)
 * [Monthly files](https://cmu.box.com/s/vh4gs3j541tt9pqn2pn72bktu0op8tpo)
 
-Please also take a look at the [individual response data](./survey-files.md),
-and the [coarse aggregates](../api/covidcast-signals/fb-survey.md) -- grouped
-only by region -- to find the data product that best suits your needs.
+These contingency tables demographic breakdowns of COVID-related topics such as
```
Comment (nmdefries) [inline docs/symptom-survey/contingency-tables.md:17]: Missing a verb ("document"? "are"?).

**Review by nmdefries — COMMENTED** [review]
Looks good!

File: docs/symptom-survey/contingency-tables.md (hunk 2 above)
Comment (capnrefsmmat) [reply docs/symptom-survey/contingency-tables.md:17]:
```suggestion
These contingency tables provide demographic breakdowns of COVID-related topics such as
```
how about this?

**Comment by capnrefsmmat** [comment]
It looks like the weekly data Box folder is currently empty; when do you think that will get populated and we can start advertising the data?

**Comment by nmdefries** [comment]
Whoops, I'll let you know when the weekly files are ready.

**Comment by nmdefries** [comment]
Weekly tables are now on Box!

**Pull request merged**